packages/api/cms-api/src/user-permissions/entities/user-permission.entity.ts

```diff
@@ -0,0 +1,50 @@
+import { BaseEntity, Entity, PrimaryKey, Property } from "@mikro-orm/core";
+import { Field, ID, ObjectType, registerEnumType } from "@nestjs/graphql";
+/* eslint-disable @typescript-eslint/naming-convention */
```

Comment:
Why do you need to disable this (for the entire file)?

Reply:
0c51de4001a90b94584112448bb6081f2acd2ba7